simapp/app.go

```diff
@@ -239,11 +237,8 @@ func NewSimApp(
 	// not include this key.
 	memKeys := sdk.NewMemoryStoreKeys(capabilitytypes.MemStoreKey, "testingkey")
 
-	// load state streaming if enabled
-	if _, _, err := streaming.LoadStreamingServices(bApp, appOpts, appCodec, keys); err != nil {
-		fmt.Printf("failed to load state streaming: %s", err)
-		os.Exit(1)
-	}
+	// register streaming services
+	RegisterStreamingServices(bApp, appOpts, keys)
```

Comment:
This `RegisterStreamingServices` resides in `test_helpers.go`. if it's going to be called in here, it should probably be somewhere else.

Reply:
It was requested that I move that into a helper function (ref: https://github.com/cosmos/cosmos-sdk/pull/14207#discussion_r1065912477). I put it in with the rest of the helper functions `simapp/app.go` uses from `test_helpers.go`. If you have a better place for it let me know.

Reply:
I'm not entirely sure where the best place for it is. But the `test_helpers.go` file is only for stuff to help with unit tests. You're calling it from `NewSimApp` though, which is very much NOT a unit test.